http://bugs.gentoo.org/462784

udiskslinuxfilesystem.c:1294:11: error: implicit declaration of function 'WIFEXITED' [-Werror=implicit-function-declaration]
udiskslinuxfilesystem.c:1294:11: error: implicit declaration of function 'WEXITSTATUS' [-Werror=implicit-function-declaration]

--- src/udiskslinuxfilesystem.c
+++ src/udiskslinuxfilesystem.c
@@ -22,6 +22,7 @@
 #include <glib/gi18n-lib.h>
 
 #include <sys/types.h>
+#include <sys/wait.h>
 #include <pwd.h>
 #include <grp.h>
 #include <string.h>
